helm/app-operator/templates/configmap.yaml

```diff
@@ -20,10 +20,11 @@ data:
         apps:
           defaultTeam: "batman"
           teams: |
+            atlas: "g8s-efk,g8s-grafana,g8s-oauth2-proxy,g8s-prometheus,gatekeeper-app,gatekeeper-control-plane-rules"
+            biscuit: "cluster-service,dex-app,g8s-cert-manager,kubernetesd,organization-operator,rbac-operator,tokend,userd,vault-app"
```

Comment:
Do we own cluster-service and kuberetesd? I think vault-exporter is missing.

Reply:
Thanks I'll check against the apps dashboard and see if I've missed any.

cluster-service and kubernetesd are both in deprecated components but no team. So not sure. If I remove them it will default to batman.

https://intranet.giantswarm.io/docs/product/team-ownership/#deprecated-components

Reply:
I added vault-exporter and removed route53-manager. I'll do a separate PR for KAAS as some other apps are missing.

Reply:
`api` is also missing. Should it also be added to biscuit? @JosephSalisbury do you have opinion here?

Reply:
I'd be fine with it if we need a decision now, but i'll see if we can find a proper ownership - see https://github.com/orgs/giantswarm/projects/10#card-43276660 for product board tracking

Reply:
Cool, let's go with this then.

Reply:
> I'd be fine with it if we need a decision now, but i'll see if we can find a proper ownership

@JosephSalisbury @kopiczko Thanks, much appreciated. If this changes let us know and we'll update.